libshaderc/include/shaderc.h

```diff
@@ -141,6 +142,56 @@ void shaderc_compile_options_set_disassembly_mode(
 void shaderc_compile_options_set_forced_version_profile(
     shaderc_compile_options_t options, int version, shaderc_profile profile);
 
+// When including a file, libshaderc initiates a query to get the full path
```

Comment:
You are right. I saw the comments in glslang::Includer say 'include()' returns the 'full path' and 'content', so I use 'fullpath' here. I think we can change the variable name to anything make sense. It seems that Glslang doesn't require the 'full path' to have any specific meaning.